### ISSUE TITLE
Iconize Icon Request

### ISSUE THREAD

**Issue by Ac5000** [FlorianWoelki/obsidian-iconize#322]
**Is your feature request related to a problem? Please describe.**
I am making a note about Iconize in my Obsidian and realized there's not an Iconize icon to use for the note! (At least not that I've seen. Please correct me if I am wrong!) I would have expected a plugin about icons would have one to choose for itself! :P

**Describe the solution you'd like**
I would like an Iconize icon/logo that I could use to decorate my Iconize related notes.

**Describe alternatives you've considered**
I have considered, and am currently using, an icon from one of the packs to substitute the missing Iconize icon.

**Additional context**
I am open to an Iconize icon being in one of the "predefined icon packs". But it would also be nice if it was somehow baked in to the plugin by default.

**Comment by FlorianWoelki** [comment]
So, you mean an icon for Iconize? :D That doesn't exist at the moment, correct. I am not sure if it makes sense to design one, but I am open for suggestions.

**Comment by Ac5000** [comment]
> So, you mean an icon for Iconize? :D

Correct! It's a pretty low priority / not super important request. It just seemed slightly ironic that a plugin about adding icons to stuff didn't have its own to use when being referenced 😄.

I suppose this means you'd need some sort of a logo / graphic to start with. Unfortunately, I am not very creative, nor artistic, or I'd try to help give suggestions.

Hopefully someone will see this though and have something that could be used and added to one of the icon packs.

Also, I won't be offended if this is too low of priority/importance for you to bother with and you'd like to close the issue. I will understand.

**Comment by FlorianWoelki** [comment]
> > So, you mean an icon for Iconize? :D
> 
> Correct! It's a pretty low priority / not super important request. It just seemed slightly ironic that a plugin about adding icons to stuff didn't have its own to use when being referenced 😄.
> 
> I suppose this means you'd need some sort of a logo / graphic to start with. Unfortunately, I am not very creative, nor artistic, or I'd try to help give suggestions.
> 
> Hopefully someone will see this though and have something that could be used and added to one of the icon packs.
> 
> Also, I won't be offended if this is too low of priority/importance for you to bother with and you'd like to close the issue. I will understand.

It's alright. I'll take care of this one whenever I've got time :)
